### PR TITLE
Obtain model if not provided from default models folder

### LINKED ISSUE #14: Add multi-language support
1) VirtualAlexa.Builder should have a `locale(string)` method for setting the locale
It will default to en-US if not set.
The value will be used in the JSON requests:
https://github.com/bespoken/virtual-alexa/blob/master/src/SkillRequest.ts#L204

2) If no interactionModel or intentSchema is supplied, VirtualAlexa should default to looking for things according to the ASK CLI format
It should check first if a models directory exists. 
If it does and there is no locale specified, it should look for models/en-US.json (the default locale)
If locale is specified, it should use the locale to lookup the correct model.

If the models directory does not exist, the existing error prompting the user to specify an intentSchema or interactionModel should be shown

### PR DESCRIPTION
**Pull request by jperata**
Closes #14

### CODE REVIEW

File: src/InteractionModel.ts [hunk 1]
```diff
@@ -59,6 +59,14 @@ export class InteractionModel {
...
     }
 
+    public static fromLocale(locale: string): InteractionModel {
+        const modelPath = "./models/" + locale + ".json";
+        if (!fs.existsSync(modelPath)) {
+            return undefined;
+        }
+
+        return InteractionModel.fromFile(modelPath);
+    }
```
Comment (jkelvie) [inline src/InteractionModel.ts]: Missing a space here after the closing bracket

File: src/VirtualAlexa.ts [hunk 2]
```diff
@@ -224,11 +226,13 @@ export class VirtualAlexaBuilder {
             const utterances = SampleUtterances.fromFile(this._sampleUtterancesFile);
             model = new InteractionModel(schema, utterances);
         } else {
-            throw new Error("Either an interaction model or intent schema and sample utterances must be provided.");
+            model = InteractionModel.fromLocale(locale);
+            if (!model) {
```
Comment (jkelvie) [inline src/VirtualAlexa.ts]: Add an additional sentence. "Alternatively, if you specify a locale, Virtual Alexa will look automatically check for the interaction model under the directory "./models" - e.g., "./models/en-US.json""

**Comment by jperata** [comment]
@jkelvie Tests for Japanese make sense as part of this one, will add them with the changes.

**Comment by jperata** [comment]
Comments addresed, Japanese support will be tested and if necesary modified in #17

**Pull request merged**